src/Goerli-DssSpell.sol

```diff
@@ -42,101 +42,29 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
-        DssExecLib.setIlkMaxLiquidationAmount("LINK-A"   ,  3 * MILLION);
-        DssExecLib.setIlkMaxLiquidationAmount("YFI-A"    ,  1 * MILLION);
-        DssExecLib.setIlkMaxLiquidationAmount("RENBTC-A" ,  2 * MILLION);
-
-        // tip changes (Max keeper incentive in DAI)
-        DssExecLib.setKeeperIncentiveFlatRate("ETH-A"           , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("ETH-B"           , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("ETH-C"           , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("WBTC-A"          , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("WBTC-B"          , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("WBTC-C"          , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("WSTETH-A"        , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("WSTETH-B"        , 250);
-        //DssExecLib.setKeeperIncentiveFlatRate("CRVV1ETHSTETH-A" , 250); // Not on Goerli
-        DssExecLib.setKeeperIncentiveFlatRate("LINK-A"          , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("MANA-A"          , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("MATIC-A"         , 250);
-        DssExecLib.setKeeperIncentiveFlatRate("RENBTC-A"        , 250); 
-        DssExecLib.setKeeperIncentiveFlatRate("YFI-A"           , 250);
-
-        // dog Hole change (Max concurrent global liquidation value)
-        DssExecLib.setMaxTotalDAILiquidationAmount(70 * MILLION);
-
-        // ---------------------------------------------------------------------
-        // MOMC Parameter Changes
-        // https://vote.makerdao.com/polling/QmbLyNUd#poll-detail
-        // https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-29-september-2022/18143
-        
-        // CRVV1ETHSTETH-A stability fee change (2.0% --> 1.5%) // Not on Goerli
-        //DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", ONE_FIVE_PCT_RATE, true); // Not on Goerli
-        
-        // YFI-A DC IAM line change (25M --> 10M)
-        DssExecLib.setIlkAutoLineDebtCeiling("YFI-A" , 10 * MILLION);
-
         // ---------------------------------------------------------------------
-        // Delegate Compensation - September 2022 // Not on Goerli
+        // Includes changes from DssSpellCollateralAction
+        // onboardCollaterals();
+        updateCollaterals();
+        // offboardCollaterals();
 
         // ---------------------------------------------------------------------
+        // Vote: https://vote.makerdao.com/polling/QmYffkvR#poll-detail
+        // Forum: https://forum.makerdao.com/t/signal-request-change-psm-gusd-a-parameters/18142
+        address MCD_PSM_GUSD_A = DssExecLib.getChangelogAddress("MCD_PSM_GUSD_A");
+        DssExecLib.setIlkAutoLineParameters({
+            _ilk:    "PSM-GUSD-A",
+            _amount: 500 * MILLION,
+            _gap:    50 * MILLION,
+            _ttl:    24 hours
+        });
+        DssExecLib.setValue(MCD_PSM_GUSD_A, "tout", 2 * WAD / 1000);
```

Comment:
Any reason we don't do it like this for readability?

```suggestion
        DssExecLib.setValue(MCD_PSM_GUSD_A, "tout", 20 * WAD / 100_00);
```

Up to you, especially wrt using 100_00 instead of 10000.

Reply:
I took this pattern from a different spell.  My gut says not to add a 0 to both

Reply:
There is no overflow risk I am seeing (evaluation of the value is determined only by constants) and it is more readable with 20 * WAD (representing 20bps), but I'll leave this up to you as it is non-blocking.